.github/workflows/release.yml

```
@@ -229,3 +229,25 @@ jobs:
           push: true
           tags: parseable/parseable:${{ github.ref_name }}
           platforms: linux/amd64,linux/arm64
+      
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)